code/modules/events/gravity_generator_blackout.dm

```diff
@@ -0,0 +1,26 @@
+/datum/round_event_control/gravity_generator_blackout
+	name = "Gravity Generator Blackout"
+	typepath = /datum/round_event/gravity_generator_blackout
+	weight = 30
```

Comment:
this has triple the chance of triggering than the default, are you sure we want this?

Reply:
It's the same weight as Tcoms blackout which I think it is in a nice middle ground of frequency from personal experience.